### PR TITLE
fix: improve accessibility of datepicker and toggle

### PR DESCRIPTION
**Pull request by mhaertwig**
- allow defining aria-labels for nextDay and prevDay buttons in datepicker
- use aria-checked for role "radio" in toggle, since aria-expanded is not allowed for this role. (see https://www.w3.org/TR/wai-aria-1.1/#radio)
- add aria-label for chips drag and drop example

### CODE REVIEW

File: src/angular/datepicker/datepicker/datepicker.ts [hunk 1]
```diff
@@ -212,6 +212,12 @@ export class SbbDatepicker<D> implements OnDestroy {
     );
   }
 
+  /** The next day button's aria-label */
+  @Input() nextDayAriaLabel: string;
```
Comment (kyubisation) [inline src/angular/datepicker/datepicker/datepicker.ts:216]: Add default values with the following pattern:
```ts
typeof $localize === 'function'
      ? $localize`:Description@@sbbDatepicker...:...`
      : '...'
```

File: src/components-examples/angular/datepicker/datepicker-simple-reactive/datepicker-simple-reactive-example.html [hunk 2]
```diff
@@ -1,6 +1,8 @@
 <sbb-form-field label="Datepicker">
   <sbb-datepicker
     [arrows]="arrows.value"
+    nextDayAriaLabel="Select next date"
```
Comment (kyubisation) [inline src/components-examples/angular/datepicker/datepicker-simple-reactive/datepicker-simple-reactive-example.html:4]: Maybe move these examples to the date range examples

**Review by kyubisation — CHANGES_REQUESTED** [review]
Thanks for the contribution 👍 
As discussed

**Comment by mhaertwig** [comment]
Split into #1014 and #1015.

**Pull request closed without merging**